src/js/helpers/session_storage.js

```diff
@@ -0,0 +1,25 @@
+"use strict";
+
+var default_cookies = require('./cookies');
+
+module.exports = Object.assign({}, default_cookies, {
+
+  set: function(name, value, minutes, domain, excl_subdomains) {
+    // Don't break the build
+    excl_subdomains = minutes + domain + excl_subdomains;
+    sessionStorage.setItem(this.encodeData(name), value);
+  },
```

Comment:
We can just remove those unused parameters.

```suggestion
  set: function(name, value) {
    sessionStorage.setItem(this.encodeData(name), value);
  },
```